server/konnectors/edf.coffee

```diff
@@ -0,0 +1,1347 @@
+xml2js = require 'xml2js'
+request = require 'request'
+async = require 'async'
+moment = require 'moment'
+cozydb = require 'cozydb'
+File = require '../models/file'
+
+parser = new xml2js.Parser()
+builder = new xml2js.Builder headless: true
+
+# TODO :
+# Move updateOrCreate in a lib
+# multi account
+# multi contract
+
+# Models
```

Comment:
There are a lot of new models introduced here, and I fear they may be too specific? Not sure what is the standard Konnectors approach here. Typically, `Contract` is a kind of translation of a [`Subscription`](http://dev.weboob.org/api/capabilities/bill) model in Weboob, but this one is much more specific.